index.html

```diff
@@ -34,6 +34,24 @@
     </div>
 </section>
 
+<section id="la-pull-request">
+    <h1>C'est quoi une PR ?</h1>
```

Comment:
- Picky mode on -
Think it should be an H2 and I would like better something like `Pull Request` as title
- Picky mode off -